hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapEntrySetOperation.java

```diff
@@ -39,7 +39,7 @@ public void run() {
         entrySet = recordStore.entrySetData();
         if (mapContainer.getMapConfig().isStatisticsEnabled()) {
             ((MapService) getService()).getMapServiceContext()
-                    .getLocalMapStatsProvider().getLocalMapStatsImpl(name).incrementOtherOperations();
+                    .getLocalMapStatsProvider().getInstantLocalMapStats(name).incrementOtherOperations();
```

Comment:
Can you reduce the size of this train wreck? I know it is just a rename, but I hate reading a.b().c().d().f()